styles.css

```diff
@@ -0,0 +1,157 @@
+*{box-sizing: border-box;}
+body{margin: 0;}
+
+.comtainer{
+    padding: 4px;
+    border: 1px solid rgb(2, 0, 10);
+    font-family: sans-serif;
+}
+
+.gap-sm{
+    gap: 8px;
+}
+
+.gap-md{
+    gap: 16px;
+}
+
+.gap-lg{
+    gap: 32px;
+}
+
+.center-wrapper{
+    max-width: 960px;
+    padding-left: 160px;
+    padding-right: 16px;
+    margin: 0 auto;
+    margin-left: 100px;
+}
+
+.grid-left{
+    grid-area: left;
+}
+
+.grid-center{
+    grid-area: center;
+    margin-left: 194px;
+    margin-top: -5px;
+}
+
+.grid-right{
+    grid-area: right;
+    margin-top: -5px;
+}
+
+.box{
+    height: 700px;
+     width: 400px;
+    padding: 4px;
+    border: 1px solid lightgrey;
+    display: grid;
+    grid-template-areas: 'left center right';
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.box-two.{
+    height: 700px;
+    width: 500px;
+}
+
+.inner-box{
+    min-width: auto;
+    min-height: auto;
+    padding: 4px;
+    border: 1px solid white    ;
+    max-width: auto;
+    max-height: auto;
+}
+
+.icon-text{
+    padding-left: 30px;
+    padding-right: 16px;
+    padding-top: 0px;
+    padding-bottom: 0px;
+    font-family: sans-serif;
+    font-weight: bolder;
+    font-size: 14px;
+}
+
+.logo{
+    width: 26px;
+    height: 26px;
+}
+
+.title{
+    font-family: sans-serif;
+    font-weight: bolder;
+}
+
+.tweet-button{
+    background-color: rgb(45, 116, 209);
+    border: none;
+    color: white;
+    border-radius: 200px;
+    display: inline-block;
+    padding: 4px 50px;
+}
+
+.tweet-button-two{
+    background-color: rgb(45, 116, 209);
+    border: none;
+    color: white;
+    border-radius: 200px;
+    display: inline-block;
+    padding: 4px 4px;
+    float: right;
+}
+
+.menu-gap{
+    min-height: 100px;
+    min-width: 200px;
+}
+
+.hr{
+    width: 394px;
+    min-width: 100%;
+}
+
+.header{
+    height: 50px;
+    width: 394px;
+    border: 1px solid white;
+}
+
+.star-icon{
+    float: right;
+}
+
+.messi-icon{
+    float: left;
+}
+
+.whats-happening{
+    margin-left: 40px;
+    color: rgb(145, 145, 145);
+}
+
+.bar-icons{
+    margin-right: 16px;
+    margin-left: 10px;
+}
+
+.button{
+    background-color: rgb(45, 116, 209);
+}
+
+.button:hover {
```

Comment:
good uses for the hover. you could also add the visted style onto your h2's in your grid-left class